### PR TITLE
Pods 5758

### PR DESCRIPTION
**Pull request by Deep547**
RacDac changes

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -230,7 +230,8 @@ urls {
     continueSchemeRegistration = "http://localhost:8200/register-pension-scheme/continue-registration"
     viewSchemeDetails = "http://localhost:8200/register-pension-scheme/scheme-details/%s"
     registerScheme = "http://localhost:8200/register-pension-scheme/what-you-will-need-before-start"
-    deleteSubscription = "http://localhost:8200/register-pension-scheme/discard"
+    deleteSubscription = "http://localhost:8200/register-pension-scheme/discard-scheme"
+    deleteSubscriptionRacdac = "http://localhost:8200/register-pension-scheme/discard-racdac"
```
Comment (madhuranakate) [inline conf/application.conf:234]: Have these urls been added to app-config-base?

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.4.9
+sbt.version=1.3.13
```
Comment (madhuranakate) [inline project/build.properties:1]: why is sbt downgraded?

**Review by madhuranakate — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (Deep547) [reply conf/application.conf:234]: That's the PR that has been raised for this:
https://github.com/hmrc/app-config-base/pull/6004

File: project/build.properties (hunk 2 above)
Comment (Deep547) [reply project/build.properties]: Actually one of us were having issues running the repo on local, so had to downgrade sbt and unfortunately it got pushed along. I have undone this change.

Comment (brownbonnie) [reply project/build.properties]: Please also double check the BSP file has been deleted if one is generated once you upgrade

**Review by madhuranakate — APPROVED** [review]

**Pull request merged**